backend/api/tests/test_utils.py

```
@@ -159,23 +159,33 @@ def test_record_user_view_idempotence(self, mock_video_view):
+
         # Assert get_or_create was called twice with the same parameters
         self.assertEqual(mock_get_or_create.call_count, 2)
-        first_call = mock_get_or_create.call_args_list[0]
-        second_call = mock_get_or_create.call_args_list[1]
-        self.assertEqual(first_call[1]['video'], second_call[1]['video'])
-        self.assertEqual(first_call[1]['viewer'], second_call[1]['viewer'])
+        _, first_call_kwargs = mock_get_or_create.call_args_list[0]
+        _, second_call_kwargs = mock_get_or_create.call_args_list[1]
+
+        # Check kwargs are the same for both calls
+        self.assertEqual(first_call_kwargs['video'], mock_video)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)